src/components/TodoTitleField/TodoTitleFields.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {
+  FC,
+  memo,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+
+interface Props {
+  oldTitle: string,
+  cancelEditing: () => void,
+  updateTitle: (title: string) => Promise<void>,
+  deleteTodoById: () => Promise<void>,
+}
+
+enum Keyboard {
+  Escape = 'Escape',
+}
```

Comment:
This enum is not belongs to the component, move it away in separated file, so you will be able to reuse it throughout you project